presto-main/src/main/java/com/facebook/presto/sql/planner/CanonicalPlanGenerator.java

```diff
@@ -830,6 +833,25 @@ public Optional<PlanNode> visitAggregation(AggregationNode node, Context context
         return Optional.of(canonicalPlan);
     }
 
+    @Override
+    public Optional<PlanNode> visitSequence(SequenceNode node, Context context)
```

Comment:
Add a test case where the sequence node is also canonicalized?

Reply:
The end to end test in testHistoryBasedStatsCalculatorCTE should suffice to make sure that it works.